src/common/commonutils/FileUtils.c

```diff
@@ -1199,6 +1198,32 @@ int CheckTextIsNotFoundInFile(const char* fileName, const char* text, char** rea
     return result;
 }
 
+static bool IsValidGrepCharacter(char text)
+{
+    return ((0 == isalnum(text)) && ('_' != text) && ('-' != text) && ('.' != text)) ? false : true;

```

Comment:
Function comments as to what is a valid grep character would be helpful.

Reply:
That's not a general grep thing, Is just for the grep command in CheckMarkedTextNotFoundInFile. Why a static here. Line 1203 says all that's needed.